docs/book/src/reference/pprof-tutorial.md

```diff
@@ -0,0 +1,62 @@
+# Monitoring Performance with Pprof
+
+[Pprof][github], a Go profiling tool, helps identify performance bottlenecks in areas like CPU and memory usage. It's integrated with the controller-runtime library's HTTP server, enabling profiling via HTTP endpoints. You can visualize the data using go tool pprof. Since [Pprof][github] is built into controller-runtime, no separate installation is needed. [Manager options][manager-options-doc] make it easy to enable pprof and gather runtime metrics to optimize controller performance.
+
+<aside class="note warning">
+<h1>Pprof Not Recommended for Production</h1>
+
+While [Pprof][github] is an excellent tool for profiling and debugging, it is not recommended to leave it enabled in production environments. The primary reasons are:
+
+1. **Security Risk**: The profiling endpoints expose detailed information about your application's performance and resource usage, which could be exploited if accessed by unauthorized users.
+2. **Overhead**: Continuous profiling can introduce performance overhead, especially under heavy load, potentially impacting production workloads.
```

Comment:
```suggestion
2. **Overhead**: Continuous profiling can introduce performance overhead mainly regards CPU usage, especially under heavy load, potentially impacting production workloads.
```

@zqzten WDYT?

Reply:
@TAM360

Reply:
`Continuous` -> `Running` might be better to clarify that CPU profiling would only run on calling the API and is not a continuous process.

Reply:
@zqzten Continuous provides better clarity.